src/pages/zh-cn/core-concepts/partial-hydration.md

```diff
@@ -42,17 +44,27 @@ import MyReactComponent from '../components/MyReactComponent.jsx';
 
 **部分激活是 Astro 在不人为进行优化情况下保持高效性能的秘密。**
 
-## 区域结构
+## 群岛架构
 
 **群岛架构**是用部分激活来构建整个网站的想法。群岛架构是对网站构建成必须分发给用户客户端的 JavaScript 捆绑包这一常见过程的替代。
 
 > 群岛架构的总体思路看似简单：在服务器上渲染 HTML 页面，并在高度动态的区域周围注入占位符或插槽。
->
-> [Islands Architecture: Jason Miller](https://jasonformat.com/islands-architecture/)
+> <br/> -- [Islands Architecture: Jason Miller](https://jasonformat.com/islands-architecture/)
 
 除了向浏览器发送较少 JavaScript 而具有明显性能优势外，群岛架构还有两个关键的好处：
 
 - **单独加载组件**。轻量级的组件（如侧边栏的切换器）将使得加载和渲染速度极快，且不会受页面中较重的组件所影响。
 - **独立组件渲染**。页面的每一部分都是一个独立的单元，一个单元的性能问题不会直接影响其他单元。
 
-![diagram](https://res.cloudinary.com/wedding-website/image/upload/v1596766231/islands-architecture-1.png)
+<IslandsDiagram>
+    <Fragment slot="headerApp">Header "app"</Fragment>
+    <Fragment slot="sidebarApp">Sidebar "app"</Fragment>
+    <Fragment slot="main">
+        Server-rendered HTML content like text, images, etc.
+    </Fragment>
+    <Fragment slot="carouselApp">Image carousel "app"</Fragment>
+    <Fragment slot="advertisement">Advertisement<br/>(server-rendered)</Fragment>
+    <Fragment slot="footer">Footer (server-rendered HTML)</Fragment>
+</IslandsDiagram>
```

Comment:
Just pointing out that you can translate this component's text if it makes sense to translate it in your language.